book/src/development/infrastructure/release.md

```diff
@@ -7,112 +7,111 @@ Clippy is released together with stable Rust releases. The dates for these
 releases can be found at the [Rust Forge]. This document explains the necessary
 steps to create a Clippy release.
 
-1. [Remerge the `beta` branch](#remerge-the-beta-branch)
-2. [Update the `beta` branch](#update-the-beta-branch)
-3. [Find the Clippy commit](#find-the-clippy-commit)
-4. [Tag the stable commit](#tag-the-stable-commit)
-5. [Update `CHANGELOG.md`](#update-changelogmd)
-
-> _NOTE:_ This document is for stable Rust releases, not for point releases. For
-> point releases, step 1. and 2. should be enough.
+1. [Defining Remotes](#defining-remotes)
+1. [Bump Version](#bump-version)
+1. [Find the Clippy commit](#find-the-clippy-commit)
+1. [Update the `beta` branch](#update-the-beta-branch)
+1. [Update the `stable` branch](#update-the-stable-branch)
+1. [Tag the stable commit](#tag-the-stable-commit)
+1. [Update `CHANGELOG.md`](#update-changelogmd)
```

Comment:
```suggestion
1. [Defining Remotes](#defining-remotes)
2. [Bump Version](#bump-version)
3. [Find the Clippy commit](#find-the-clippy-commit)
4. [Update the `beta` branch](#update-the-beta-branch)
5. [Update the `stable` branch](#update-the-stable-branch)
6. [Tag the stable commit](#tag-the-stable-commit)
7. [Update `CHANGELOG.md`](#update-changelogmd)
```

Reply:
I went with that, so when inserting a new chapter in the future, you don't have to update all the following numbers. But happy to change that.